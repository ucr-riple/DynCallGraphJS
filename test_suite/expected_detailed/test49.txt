{
    'global (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test49.js:1:1:19:5)': [
        'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test49.js:1:2:19:2)'
    ],
    'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test49.js:17:9:17:26)': [
        'name (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test49.js:10:1:12:2)'
    ]
}