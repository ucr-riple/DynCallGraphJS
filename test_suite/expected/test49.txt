{ '(test49.js:1:1:19:5)':
   [ '(test49.js:1:2:19:2)' ],
  '(test49.js:17:9:17:26)':
   [ '(test49.js:10:1:12:2)' ] }